packages/luna/src/components/TutorFeedback/RatingDialog.tsx

```diff
@@ -77,7 +77,10 @@ export const RatingDialog: React.FC<RateDialogProps> = ({
         <Button
           size={ButtonSize.Large}
           onClick={() =>
-            onSubmit({ value: newRating, feedback: newFeedback || null })
+            onSubmit({
+              value: newRating,
+              feedback: newFeedback,
```

Comment:
It is important to put `null` here because the feedback can be any empty string which might cause the backend to response with error. 

```suggestion
              feedback: newFeedback || null,
```